odoo/addons/base/models/ir_model.py

```diff
@@ -2212,7 +2213,8 @@ def _process_end(self, modules):
             if record.exists():
                 module = xmlid.split('.', 1)[0]
                 record = record.with_context(module=module)
-                if record._name == 'ir.model.fields' and not module.startswith('test_'):
+                if record._name == 'ir.model.fields' and not module.startswith('test_') \
```

Comment:
The main question here, shouldn't we just remove this warning since upgrade makes the same test. I need to check if it is the case every time

Reply:
Indeed!

Reply:
Done